dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/RefinementSolver.java

```diff
@@ -622,23 +623,32 @@ private boolean process(Constraint c) {
         Any larger choice for b' will make r smaller, but since r is non-positive this reduces the chances
         of satisfying negated axioms (e.g., if "~empty(r)" is true for some small r, then it is also true for all larger r).
         On the other hand, if b' is chosen too small, then r gets so large that it violates "empty(r & b)".
-        We prove this formally:
+
+        We prove this formally. Towards this, we reason about executions M which are to be understood
+        as an interpretation of the relations of the memory model.
+        So, e.g., M(rf) is the value of rf (=the concrete binary relation) in the execution M.
 
         "NEW => ORIGINAL":
-        Suppose M' is a consistent execution of the new memory model.
-        Then from "let r = a \ b'" it follows that M'(r) \subseteq M'(a).
-        Furthermore, from "empty (r & b)" it follows that M'(r) \subseteq ~M'(b).
-        Combining both, we get M'(r) \subseteq M'(a) & ~M'(b) = M'(a) \ M'(b)
-        Now consider an execution M of the original memory model that matches with M' on the base relations
+        Suppose M' is a consistent execution under the new memory model.
+        Then from "let r = a \ b'" it follows that
+            M'(r) \subseteq M'(a).
+        Furthermore, from "empty (r & b)" it follows that
+            M'(r) \subseteq ~M'(b).
+        Combining both, we get
+            M'(r) \subseteq (M'(a) & ~M'(b)) = (M'(a) \ M'(b)).
+        Now consider an execution M under the original memory model that matches with M' on the base relations
```

Comment:
"execution" -> "candidate execution"

Reply:
Why do you prefer the term candidate execution? I think we never use this anywhere. I added an introductory sentence explaining what an execution of a memory model is in-line with our usage of that term in the two OOPSLA papers.

Reply:
Most people would parse "execution" as "consistent executions", thus is why I normally prefer to use "candidate execution" or "behavior" so people don't understand it wrong.

This is just a pedantic comment from my side. As you said, the comment explains how executions should be interpret.